src/backends/torch/torchUtils.h

```diff
@@ -0,0 +1,47 @@
+#ifndef TORCH_UTILS_H
+#define TORCH_UTILS_H
+
+#if !defined(CPU_ONLY)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-parameter"
+#include <c10/cuda/CUDACachingAllocator.h>
+#pragma GCC diagnostic pop
+#endif
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-parameter"
+#include <torch/torch.h>
+#pragma GCC diagnostic pop
+#include <torch/script.h>
+
+namespace dd
+{
+
+  namespace torch_utils
+  {
+    inline void empty_cuda_cache()
```

Comment:
This appears unused ? Also, we may need a warning comment on functions that affect the global CUDA state.